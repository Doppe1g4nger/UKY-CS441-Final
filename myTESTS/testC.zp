 int main ()
{
    int i;
    for (i; i < 5; i = i + 1)
    {
        int var;
        var = getnum();
        if (var < 15)
        {
            puts("Less Than 15\n");
        }
        else if (var < 25)
        {
            puts("Less Than 25\n");
        }
        else
        {
            puts("Greater than or equal to 25\n");
        }
    }
}
