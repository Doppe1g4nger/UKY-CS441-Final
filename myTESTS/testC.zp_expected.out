Less than 15
Less than 15
Less than 25
Less than 25
Greater than or equal to 50
